### PR TITLE
BUILD(cmake): Install overlay libs and scripts if enabled

### PR DESCRIPTION
**Pull request by DarkDefender**
Before the overlay libraries and the related script would not be
installed. Instead the user would have to manually install them on the
system.

Now the overlay libraries are added to the install. The script was fixed
to take into account the correct overlay library names.

Closes: #4471

### CODE REVIEW

**Comment by DarkDefender** [comment]
Sadly I can't actually check if the overlay is working correctly as I get segfaults when trying to use it.

For example `mumble-overlay glxgears` segfaults in /usr/lib64/libGLdispatch.so which belongs to `libglvnd`.
Perhaps this is because I'm using Xwayland (sway) instead of native Xorg?

Is someone could double check that this actually works, it would be great.

File: overlay_gl/CMakeLists.txt [hunk 1]
```diff
@@ -91,6 +91,8 @@ if(NOT APPLE)
 					"-ldl"
 					"-lrt"
 			)
+      # install 32bit overlay library
+      install(TARGETS overlay_gl_x86 LIBRARY DESTINATION "${MUMBLE_INSTALL_LIBDIR}")
 		endif()
```
Comment (Krzmbrzl) [inline overlay_gl/CMakeLists.txt:96]: The indentation seems to be off (we indent using tabs)

**Review by Krzmbrzl — CHANGES_REQUESTED** [review]
I think it would be a bit cleaner if you separated your changes into 2 distinct commits:
1. Fixing the `mumble-overlay` script
2. Setting the install dirs

File: overlay_gl/CMakeLists.txt (hunk 1 above)
Comment (DarkDefender) [reply overlay_gl/CMakeLists.txt:96]: Ah, my bad. This is fixed now.

Comment (Krzmbrzl) [reply overlay_gl/CMakeLists.txt:96]: No it's not. It is still indented using spaces xD

Comment (DarkDefender) [reply overlay_gl/CMakeLists.txt]: Huh? The changes must have gotten lost somehow when I split the commit in two.
Third times the charm :crossed_fingers:

Comment (Krzmbrzl) [reply overlay_gl/CMakeLists.txt]: Yes indeed :D

**Review by Krzmbrzl — APPROVED** [review]

**Comment by Krzmbrzl** [comment]
> Perhaps this is because I'm using Xwayland (sway) instead of native Xorg?

That could very well be.

I think we'll just assume this to be working on "normal" Xorg systems and go ahead and merge this PR. If it doesn't work there either, I'm sure we'll get reports about it at some point :D

**Comment by Krzmbrzl** [comment]
Thank you for your contribution :+1:

**Pull request merged**